.github/workflows/run-tests.yml

```diff
@@ -230,9 +230,10 @@ jobs:
           - postgresql
           - sqlite
           - firebird
-          #- snowflake
+          - snowflake
     needs: build_tests
     timeout-minutes: 30
+    if: ${{ matrix.testSystem != 'snowflake' }} 
```

Comment:
This is still going to prevent the tests from running on Snowflake on master.

Reply:
I think we can apply the same logic like in the below ones, what do you think?
`if: ${{ github.event_name == 'push' && github.ref == 'refs/heads/master' && matrix.testSystem == 'snowflake' }}`

Reply:
That doesn't look right to me. You need to conditionally skip the tests run on line 295: https://github.com/liquibase/liquibase/pull/5806/files#diff-7314d0ebbd2e9537ae4889316745b4fd2fa43cb86275c9caae18a86ba228b642R295 when the branch is not master and the test system is snowflake.

Reply:
@StevenMassaro : made the change. I think this needs to be merged to master to check some if it runs only on master and not on a PR. 
cc: @filipelautert